packages/eslint-config/package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Couldn't we just set it in root `eslintrc`, then extend each workspace's eslint with it, instead of having a new workspace?

Reply:
I think the new config format recommends that you have a `eslint.config.js` in each workspace to ensure that plugins are correctly resolved (especially in a pnpm setup). This internal package will ensure that we can share the same config as we add more packages.